src/sbl_filing_api/services/request_handler.py

```diff
@@ -0,0 +1,24 @@
+import logging
+
+import httpx
+from sbl_filing_api.config import settings
+
+logger = logging.getLogger(__name__)
+
+
+def send_confirmation_email(
+    user_full_name: str, user_email: str, contact_info_email: str, confirmation_id: str, timestamp: int
+):
```

Comment:
If we want to be really particular, the email fields can by pydantic EmailStr types to really force the issue.  The mail API expects those types, so might be good to expect those here before passing them along, too.  Probably overkill, but meh, why not.